FWCore/Integration/test/DetSetVectorThinningTest1_cfg.py

```diff
@@ -0,0 +1,230 @@
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process("PROD")
+
+process.options = cms.untracked.PSet(
```

Comment:
You can do just
```suggestion
process.options.numberOfStreams = 1
```

Reply:
I know, and as you can probably guess, the added files are partly copied from existing tests (the top part esepecially). The `numberOfStreams` could actually be dropped because 1 is the default, and the test should not really be sensitive to the number of threads.